packages/sdk-middleware-auth/src/base-auth-flow.js

```diff
@@ -99,10 +99,17 @@ function executeRequest({
         }
         const error: Object = new Error(parsed ? parsed.message : text)
         if (parsed) error.body = parsed
+
+        // Dispatch all pending requests
```

Comment:
I'm not understanding this comment, it looks like we are just rejecting below and not actually dispatching any requests?

Reply:
There is queue of requests,
For ex, once 1st request is failed, then requestState set to false, and dispatch all other queued requests
so 2nd request starts and so on

Reply:
I can see that we do that in the condition above: https://github.com/commercetools/nodejs/blob/192f8d02a657bc500ffb0b8845f80bcb1aeadad2/packages/sdk-middleware-auth/src/base-auth-flow.js#L82-L88

but are you sure we also keep dispatching here? I cant see it in the code so maybe I'm just missing a part of the logic somewhere.

Reply:
You right, it should be then sth like reset request status!